manifests/init.pp

```diff
@@ -54,8 +54,8 @@
 class letsencrypt (
   Optional[String] $email                = undef,
   String $path                           = $letsencrypt::params::path,
-  $venv_path                             = $letsencrypt::params::venv_path,
-  Array $environment                     = [],
+  $venv_path                             = [],
```

Comment:
This doesn't look like a correct rebase

Reply:
but now that i'm seeing this, we've already had `$venv_vars` — is this even still needed?

Reply:
the answer is: potentially, but it needs to be rethought, rather than rebased.

Reply:
No, we had `venv_path` so it's still useful to have I think